pkgs/build-support/rust/default.nix

```diff
@@ -86,6 +86,10 @@ in stdenv.mkDerivation (args // {
     ${stdenv.lib.optionalString (stdenv.buildPlatform.config != stdenv.hostPlatform.config) ''
     [target."${stdenv.hostPlatform.config}"]
     "linker" = "${ccForHost}"
+    ${stdenv.lib.optionalString (stdenv.hostPlatform.config == "aarch64-unknown-linux-musl") ''
```

Comment:
Does it have those symbols on any musl platform?
Otherwise we could set `stdenv.hostPlatform.isMusl`.

Reply:
It does have these symbols on x86_64 musl platform.

Reply:
Are you sure? I cannot find it: 

```
$ nm -gC $(nix-build --no-out-link '<nixpkgs>' -A musl)/lib/libc.so | grep __addtf3
```

I ask since I also cannot find a definition in the musl source repository.

Reply:
I assumed that's the case, but at the very least I'm absolutely sure that cross-compiled x86_64-unknown-linux-musl executables work without this change. It seems that these symbols are used by LLVM for floating point arithmetic on AArch64 only.